<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drinkfill_ev</class>
 <widget class="QMainWindow" name="drinkfill_ev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>drinkfill_ev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="companyLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>608</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="inventoryTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="curInventoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Current Inventory</string>
      </property>
      <widget class="QListView" name="curInventoryView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="updateInventoryBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>341</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Inventroy Management</string>
      </property>
      <widget class="QPushButton" name="volumeUpdateButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QComboBox" name="optionComboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="volumeEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="machineIDComboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="labelTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="productBox">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>571</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="productView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>551</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="newImageButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Product 
Image</string>
      </property>
     </widget>
     <widget class="QComboBox" name="newProductSelectBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="newProductNameEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="newProductPrice1Edit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="newProductAmountEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="newProductPrice2Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="eventLog">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="activityBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>581</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="activityView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>561</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="activityButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>260</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="customerLog">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="transactionBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="transactionView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>31</y>
         <width>561</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="transactionButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>260</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="setupTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="setupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLineEdit" name="machineIdEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="machineIdButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="numDrinksEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="numDrinksButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>130</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearProductDBButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
